pkg/controller/tiers/tiers_controller.go

```diff
@@ -133,6 +133,9 @@ func (r *ReconcileTiers) Reconcile(ctx context.Context, request reconcile.Reques
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling Tiers")
 
+	// Mark CR as found even though this controller is not associated with a CR, as OnCRFound() enables TigeraStatus reporting.
+	r.status.OnCRFound()
```

Comment:
So we were already attempting to report a tier status, this was all the prevented it from actually being reported?

Reply:
Yeah, the controller was already following the pattern of reporting a degraded status before failing the reconcile, perhaps from copy-paste. It was intentional for the controller to not report to TigeraStatus though